src/transforms/transforms.ts

```diff
@@ -51,6 +51,11 @@ export function transformChoice(
     schemaType,
     description: `Defines values for ${metadata.name}.`,
     serializedName: metadata.name,
+    names: choice.choices.map(c =>
+      c.language.javascript
```

Comment:
We have a helper that extracts the typescript or default `import { getLanguageMetadata } from "../utils/languageHelpers";`

Also, Since name and value are correlated, it may be easier to work with these pair as a single array of `{name, value}` instead of one array for value and one for names

Reply:
@joheredi Completed changes in the latest commit. Please take a look